src/server/api.ts

```diff
@@ -131,6 +131,7 @@ router.get("/logout/session", (req, res) => {
   res.clearCookie("feide_auth", { domain: `.${config.feideDomain}` });
   const state = typeof req.query.state === "string" ? req.query.state : "/";
   const { basepath, basename } = getLocaleInfoFromPath(state);
+  privateRoutes.unshift(MINNDLA);
```

Comment:
Kun for å få til utlogging til forsida fra /minndla

Reply:
Vil ikke dette mutere `privateRoutes`?

Reply:
Hadde en greiere løsning vært å sjekke om `basePath` tilsvarte `minndla`, og så redirecte hvis det stemmer?

Reply:
Vil det mutere privateroutes globalt?

Reply:
Ja, det vil jeg da tro. `unshift` muterer.

Reply:
Sant det. Eg tenkte det var en lokal kopi av arrayet, men det stemmer jo ikkje.